components/scream/tests/coupled/homme_physics/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+include (ScreamUtils)
+
+# Needed for RRTMGP
+find_library(NETCDF_C netcdf HINTS ${NetCDF_C_PATHS}/lib)
+## Add CUDA flags for YAKL
+if (CUDA_BUILD)
+    set(ARCH "CUDA")
+    set(YAKL_CXX_FLAGS "-D__USE_CUDA__ --expt-extended-lambda --expt-relaxed-constexpr ${CUDA_FLAGS} ${YAKL_CXX_FLAGS}")
+endif()
+
+# Get or create the dynamics lib
+#                 HOMME_TARGET   NP PLEV QSIZE_D
+CreateDynamicsLib("theta-l_kokkos"  4   72   35)
+
+set (NEED_LIBS cld_fraction shoc p3 scream_rrtmgp rrtmgp ${NETCDF_C} ${dynLibName} scream_control scream_share physics_share yakl)
+
+# Add source files
+set (SRC shoc_cld_p3_rrtmgp.cpp ${SCREAM_BASE_DIR}/src/physics/rrtmgp/tests/rrtmgp_test_utils.cpp)
+
+# Test Homme standalone
+if (NOT SCREAM_AUTOTESTER AND NOT EKAT_ENABLE_VALGRIND)
+  CreateUnitTest(homme_physics_stand_alone "homme_physics_stand_alone.cpp" "${NEED_LIBS}" EXE_ARGS "< namelist.nl" LABELS "dynamics;driver;shoc;cld;p3;rrtmgp;physics")
+  set_target_properties(homme_physics_stand_alone PROPERTIES COMPILE_FLAGS "${YAKL_CXX_FLAGS}")
+  target_include_directories(homme_physics_stand_alone PUBLIC
+    ${SCREAM_BASE_DIR}/src/physics/rrtmgp
```

Comment:
Doesn't the `scream_rrtmgp` target already add these folders to its include dirs? If so, these are unnecessary, since liinking(*) against `scream_rrtmgp` already takes care of propagating the include folders.

(*) Recall that, when what is linked is actually a cmake target generated in the current project (or imported),  `target_link_libraries` does more than just linking, namely propagates compiler flags/definitions, include directories, link directories, dependencies,...

Reply:
TBH, I'm not sure.  I recall adding all of these include directories after the test wouldn't work.  When I created the CMake for this test I started as lean as possible and added pieces as the build or run would fail.

Reply:
I just checked, and rrtmgp _does_ attach the include dirs to the target as PUBLIC, so there's no need to add them again here. I would remove completely the `target_include_directories` command.